app/groceries/groceries.component.ts

```diff
@@ -1,41 +1,38 @@
-import {Component, ElementRef, OnInit, Pipe, PipeTransform, ViewChild} from "@angular/core";
-import {Router} from "@angular/router";
-import {Color} from "color";
-import {action} from "ui/dialogs";
-import {Page} from "ui/page";
-import {TextField} from "ui/text-field";
-import {Config} from "../../shared/config";
-import {Grocery} from "../../shared/grocery/grocery";
-import {GroceryList} from "../list/grocery-list.component";
-import {GroceryStore} from "../../shared/grocery/grocery-list.service";
-import {alert} from "../../utils/dialog-util";
-import {setHintColor} from "../../utils/hint-util";
+import { Component, ElementRef, OnInit, ViewChild } from "@angular/core";
+import { Router } from "@angular/router";
+import { Color } from "color";
+import { action } from "ui/dialogs";
+import { Page } from "ui/page";
+import { TextField } from "ui/text-field";
```

Comment:
Per the style guide I think there should be a new line here between third-party imports and one in this project.